### PR TITLE
CFD-418: Voices & Views carousel : missing creation date

### PR DESCRIPTION
**Pull request by YuriGizra**
#588

### CODE REVIEW

**Comment by YuriGizra** [comment]
Output:
selection_129

**Comment by YuriGizra** [comment]
@nader77, ready for review.

File: capacity4more/themes/c4m/kapablo/sass/components/_carousel.scss [hunk 1]
```diff
@@ -51,8 +52,14 @@ ul[rn-carousel] {
...
           text-transform: uppercase;
           font-size: 18px;
+          padding-bottom: 0;
+        }
+
+        &.date {
+          font-weight: bold;
         }
       }
+
```
Comment (nader77) [inline capacity4more/themes/c4m/kapablo/sass/components/_carousel.scss]: Remove empty line.

File: capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module [hunk 2]
```diff
@@ -135,6 +135,7 @@ function c4m_features_homepage_voices_views_carousel() {
       'id' => $wrapper->getIdentifier(),
       'title' => $wrapper->label(),
       'text' => $wrapper->c4m_intro_text->value(),
+      'date' => format_date($wrapper->created->value(),'custom', 'F d Y'),
```
Comment (nader77) [inline capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module]: Add a space before 'custom' => `` 'custom'``

**Comment by YuriGizra** [comment]
@zero2one,  PR is ready for review. I can't assign you because I have not permissions yet.

**Pull request merged**